### PR TITLE
clean up unnecessary code

### PR DESCRIPTION
**Pull request by Pingan2017**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Pingan2017** [comment]
ping @yujuhong @vishh @pmorie

File: pkg/kubelet/images/image_manager.go [hunk 1]
```diff
@@ -60,10 +60,6 @@ func NewImageManager(recorder record.EventRecorder, imageService kubecontainer.I
 // shouldPullImage returns whether we should pull an image according to
 // the presence and pull policy of the image.
 func shouldPullImage(container *v1.Container, imagePresent bool) bool {
-	if container.ImagePullPolicy == v1.PullNever {
```
Comment (yujuhong) [inline pkg/kubelet/images/image_manager.go]: I actually like the explicit check. If we add more pull policy one day, this would still work correctly.

Comment (Pingan2017) [reply pkg/kubelet/images/image_manager.go]: Even if we add more policy, we only need to judge when the function returns true. and all others returns false immediatly. We do not need to judge when this function returns false additionally. I think there is no problem if we delete this check.

**Pull request closed without merging**